### PR TITLE
[Add] add leaky relu to actifun

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Add] add leaky relu to actifun</summary><br />

This patch add leaky relu to actifun, for now there is no need to add
slope parameter, this is more like a quick fix

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: nntrainer/layers/acti_func.cpp [hunk 1]
```diff
@@ -261,4 +269,21 @@ float ActiFunc::reluPrime(float x) {
 float ActiFunc::no_op(float x) { return x; }
 
 float ActiFunc::no_op_prime(float x) { return 1.0f; }
+
+constexpr static inline float NEGATIVE_SLOPE = 0.01f;
```
Comment (kparichay) [inline nntrainer/layers/acti_func.cpp:273]: i recommend making this a property as this is usually configurable.

File: nntrainer/layers/acti_func.h [hunk 2]
```diff
@@ -144,6 +135,28 @@ class ActiFunc {
    */
   static float no_op_prime(float x);
 
+  /**
+   * @brief leaky relu function
+   * @note slope parameter is needed for leaky relu, but supporting property on
+   * this class will need extensive refactoring. For now 0.01 is used for
+   * negative slope.
```
Comment (zhoonit) [inline nntrainer/layers/acti_func.h:142]: .

File: nntrainer/layers/acti_func.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/layers/acti_func.cpp:273]: Yes, this need be. For now this is an improvised version to support current needs. (This has to be supported by monday).

Please refer to https://github.com/nnstreamer/nntrainer/pull/1742/files#r756727752

**Review by kparichay — APPROVED** [review]
LGTM

File: nntrainer/layers/acti_func.cpp (hunk 1 above)
Comment (kparichay) [reply nntrainer/layers/acti_func.cpp:273]: cool.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**